packages/ckb-sdk-core/src/signWitnesses.ts

```diff
@@ -34,23 +34,23 @@ const signWitnesses: SignWitnesses = (key: Key | Map<LockHash, Key>) => ({
   witnesses: StructuredWitness[]
   inputCells: CachedLock[]
 }) => {
-  if (!key) throw new ArgumentRequired('Private key')
+  if (!key) throw new ArgumentRequired('Signature provider')
```

Comment:
Please update the test case

Reply:
https://github.com/nervosnetwork/ckb-sdk-js/blob/develop/packages/ckb-sdk-core/__tests__/signWitnesses/fixtures.json#L183

Reply:
https://github.com/nervosnetwork/ckb-sdk-js/blob/develop/packages/ckb-sdk-core/__tests__/signWitnesses/fixtures.json#L112

Reply:
The test case is updated and passed now.